1-exercises/B-while-loop/exercise.js

```diff
@@ -5,8 +5,24 @@
     The list of numbers should start with 0. n is being passed in as a parameter.
 */
 
-function evenNumbers(n) {
-    // TODO
+function evenNumbers(n)
+{
+    let total = "0";
+    let calc = 0;
+    
+    if (n === 0)
+    {
+        return console.log("");
+    }
+
+    for(let i = 1; i < n; i++)
+    {
+        calc = calc + 2;
+
+        total = total + ", " + calc;
+    }
+
+    return console.log(total);
```

Comment:
The instructions said that you should use a while loop.
Consider using an array to store the values and then convert it to string using 'toString' method